src/index.js

```diff
@@ -95,20 +95,13 @@ var spawn = require('child_process').spawn;
 
 var vroomCommand = args['path'] + 'vroom';
 var options = [];
-if (args['libosrm']) {
-  options.push('-l');
-} else {
-  options.push('-a', args['osrm_address']);
-  options.push('-p', args['osrm_port']);
-}
+options.push('-r', args['router']);
+options.push('-a', args['router_address']);
+options.push('-p', args['router_port']);
```

Comment:
Default values for `-a` and `-p` don't make sense when using `libosrm`, we should probably test for the content of `args['router']` to decide whether to add them or not.

Also to handle using several profiles, we'd have to pass the `-a` and `-p` several times. Typical use with OSRM with two servers running locally on different ports would be:
```
vroom -a car:0.0.0.0 -p car:5000 -a bike:0.0.0.0 -p bike:5001 [...]
```
This would also be required for ORS so that the vroom binary can bind profiles to the relevant server/port (even if they are identical), or issue an error for an unsupported profile.

I suspect minimist would override values with several command-line occurrences, so maybe we should have a hard-coded variable describing the profile/server mapping?

Reply:
Deciding on `args['router']`whether to add `-a` and `-p`makes totally sense.

I did a few tests. Apparently minimist isn't able to deal with multiple invocations of the same CL option. There is a PR substack/minimist#51 implementing this but it has not been merged within 3 years. As you propose, a hard-coded variable could also do it. What do you think about removing the options `router_address` and `router_port`and introducing two configuration variables that may be used as in the following example?
```JavaScript
// Example configuration
const routerAddresses = ["car:0.0.0.0", "bike:0.0.0.1"];
const routerPorts = ["car:5000", "bike:5001"];
```

Reply:
> What do you think about removing the options `router_address` and `router_port`

I'm fine with that. Maybe embedding everything in a single object with profiles as keys? Something like:
```javascript
var routingServers = {
  "car": {
    "host": "0.0.0.0",
    "port": 5000
  },
  "bike": {
    "host": "0.0.0.0",
    "port": 5001
  }
}
```

Reply:
Done.